tests/product/tests/data/file_system/eks_file_system_test_cases.py

```diff
@@ -80,3 +92,17 @@
     "3.1.4 Kubelet-config ownership invalid user evaluation failed": cis_eks_3_1_4_user_fail,
     "3.1.4 Kubelet-config ownership invalid group evaluation failed": cis_eks_3_1_4_group_fail,
 }
+
+cis_eks_all = {
+    "test-eks-config-1": {
+        **cis_eks_3_1_1,
+        **cis_eks_3_1_2,
+        **cis_eks_3_1_3,
+        **cis_eks_3_1_4,
+    },
+    "test-eks-config-2": {
+        **cis_eks_3_1_1_conf_2,
+    },
+}
```

Comment:
not sure I'm following here, can you add a comment to explain the format, please?

Reply:
I added comments there. But this is relevant for all test cases as well.
Just to be clear I will explain how it works:
Each rule may contain several test cases depended on configuration. The configuration is provided through environment var and known when the test execution starts. This dictionary summarizes all cases and all configurations. But during runtime only one of them may be used (test-eks-config-1 / test-eks-config-2)
When eks test cases file is loaded the following is executed
`cis_eks_file_system_cases = cis_eks_all[eks.current_config]`